static/docker-compose.yml

```diff
@@ -80,6 +80,42 @@ services:
     volumes:
     - letsencrypt:/etc/letsencrypt:rw
 
+  notification-engine:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    depends_on:
+      - plextracdb
+      - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      COUCHBASE_URL: "http://plextracdb"
+      LOG_LEVEL: "${LOG_LEVEL:-error}"
+      REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
+      REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+    entrypoint: npm run
+    command: "start:notification-engine"
+
+  notification-sender:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    depends_on:
+      - plextracdb
+      - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      COUCHBASE_URL: "http://plextracdb"
+      CLIENT_DOMAIN_NAME: ${CLIENT_DOMAIN_NAME:-err}
```

Comment:
👍

Reply:
Here's the corresponding change in the dev docker-compose: https://github.com/PlexTrac/product-core-backend/pull/575/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R75